tests/DemoBundle/Entity/Product.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2018 Spomky-Labs
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+namespace BehatExtension\DoctrineDataFixturesExtension\Tests\DemoBundle\Entity;
+
+class Product
+{
+    /**
+     * @var int
+     */
+    private $id;
+
+    /**
+     * @var string
+     */
+    private $name;
+
+    /**
+     * @var null|string
+     */
+    private $description;
+
+    /**
+     * Product constructor.
+     *
+     * @param string      $name
+     * @param null|string $description
+     */
+    public function __construct(string $name, ?string $description = null)
+    {
+        $this->name = $name;
+        $this->description = $description;
+    }
+
+    /**
+     * @return int
+     */
+    public function getId(): int
```

Comment:
not really important for fixture test entity, but id is assigned only after flushing, so it can have null here